app/main/views/buyers.py

```diff
@@ -533,8 +533,11 @@ def get_context_data(self, **kwargs):
         if brief['status'] not in CLOSED_PUBLISHED_BRIEF_STATUSES:
             abort(404)
 
-        text_type = str if sys.version_info[0] == 3 else unicode
-        filename = inflection.parameterize(text_type(brief['title']))
+        # Filename must be unicode
```

Comment:
http://pythonhosted.org/six/#six.text_type

Six can take care of this and we've got it as a dependency.

Reply:
Sweet, I didn't know we had that - much neater all round.